src/BUILD

```diff
@@ -648,7 +656,7 @@ JAVA_VERSIONS = ("11",)
         name = "create_java_tools_build_java" + java_version,
         srcs = ["//tools/jdk:BUILD.java_tools"],
         outs = ["remote_java_tools_java" + java_version + "/BUILD"],
-        cmd = "sed 's/JAVA_LANGUAGE_LEVEL/" + java_version + "/g' $< > $@",
+        cmd = "sed 's/JAVA_LANGUAGE_LEVEL/" + java_version + "/g; s/PLATFORM/" + _PLATFORM + "/g' $< > $@",
```

Comment:
My read of this is that only the remote JDK that matches the host platform will be generated. Is this the intended use?

Reply:
I updated the comment.